examples/list_adapters/list_adapters.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "simpleble/Adapter.h"
 
-int main(int argc, char* argv[]) {
+int main() {
     auto adapter_list = SimpleBLE::Adapter::get_adapters();
 
     if (adapter_list.size() == 0) {
```

Comment:
To be consistent with the other examples, we should probably return `1` instead of `0`.